be/src/service/doris_main.cpp

```diff
@@ -50,6 +50,7 @@
 #include "agent/topic_subscriber.h"
 #include "util/doris_metrics.h"
 #include "olap/options.h"
+#include "olap/storage_engine.h"
```

Comment:
Is it necessary here?

Reply:
Yes. `StorageEngine` is used in this file. 
Previously, because of some unneeded header file included in `agent_server.h` will include `storage_engine.h`, these unneeded header files were removed from `agent_server.h`.